### PR TITLE
Feature/bureau bidders

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
Bureau -> Position Manager -> Click on a position

~[pause merge](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1025)~

### CODE REVIEW

File: src/Components/MailToButton/MailToButton.test.jsx [hunk 1]
```diff
@@ -17,4 +20,18 @@ describe('MailToButtonComponent', () => {
     );
     expect(toJSON(wrapper)).toMatchSnapshot();
   });
+
+  it('text renders before icon', () => {
+    const wrapper = shallow(<MailToButton
+      textBefore={'test text'}
+    />);
+    expect(wrapper.find('a').text()).toBe('test text <FontAwesome /> ');
```
Comment (elizabeth-jimenez) [inline src/Components/MailToButton/MailToButton.test.jsx:28]: 🤮

File: src/Components/MailToButton/MailToButton.test.jsx [hunk 2]
```diff
@@ -17,4 +20,18 @@ describe('MailToButtonComponent', () => {
...
+  });
+
+  it('text renders after icon', () => {
+    const wrapper = shallow(<MailToButton
+      {...props}
+    />);
+    expect(wrapper.find('a').text()).toBe(' <FontAwesome /> test text');
```
Comment (elizabeth-jimenez) [inline src/Components/MailToButton/MailToButton.test.jsx:35]: 🤢

**Comment by mjoyce91** [comment]
@elizabeth-jimenez merge conflicts 😢

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez merge conflicts 😢

nooooooooo; may curses befall the forks!

**Comment by mjoyce91** [comment]
The font size responsiveness here feels a little weird. What if we just matched from above, regardless of screen width?

Screen Shot 2020-07-29 at 3 06 46 PM

**Comment by mjoyce91** [comment]
I like that you added the retained space - what if we commented it out for now until it gets used?

Screen Shot 2020-07-29 at 3 55 25 PM

File: src/Components/MailToButton/MailToButton.test.jsx (hunk 1 above)
Comment (mjoyce91) [reply src/Components/MailToButton/MailToButton.test.jsx:28]: There's probably some type of "contains" function

File: src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        language: 'Turkish (TU)',
+        reading_proficiency: '2',
+        spoken_proficiency: '2',
+        representation: 'Turkish (TU) 2/2',
+      },
+    ];
+
+    const name = (<Link to={'www.google.com'}>{propOrDefault(bidder, 'name')}</Link>);
+    const skills = (<SkillCodeList skillCodes={bidder.skills} />);
+    const language = (<LanguageList languages={languages} propToUse="representation" />);
```
Comment (mjoyce91) [inline src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:134]: 👍

File: src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+          </thead>
+          <tbody>
+            {/* eslint-disable-next-line no-unused-vars */}
+            {[...Array(10).keys()].map((m) => (
+              tableRows
+            ))}
+          </tbody>
+        </table>
+      </div>
+    );
```
Comment (mjoyce91) [inline src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:174]: Very clean, I like it

Comment (elizabeth-jimenez) [reply src/Components/BureauPage/PositionManagerBidders/PositionManagerBidders.jsx:174]: i stole this `[...Array(10).keys()]` from reviewing one of your PRs. I really like it!

**Pull request merged**